src/app/admin/admin.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators, ReactiveFormsModule } from '@angular/forms';
+import { MatTableDataSource } from '@angular/material';
+import { HttpService } from "../shared/http.services";
+
+
+export interface User {
+  firstName: string;
+  lastName: string;
+  email: string;
+  role: string;
+}
+
+const userRoles: String[] = ['Student',
+  'Teacher'
+];
+
+const ELEMENT_DATA: User[] = [
```

Comment:
When you integrate with my changes, these will be shifted to the mongoDB collection. Don't forget to handle the 'username' attribute. You have not included it in your json.

Reply:
Will handle userName once I work on DB side integration.